package-lock.json

```diff
@@ -3800,7 +3812,7 @@
         },
         "ansi-html": {
             "version": "0.0.7",
-            "resolved": "https://registry.npmjs.org/ansi-html/-/ansi-html-0.0.7.tgz",
+            "resolved": "https://artifactory.corp.alleninstitute.org:443/artifactory/api/npm/npm-virtual/ansi-html/-/ansi-html-0.0.7.tgz",
```

Comment:
do we want to switch these to AI artifactory?

Reply:
thank you, good catch - that should not have happened. (although the automated build seems to be working) I'll try to remove those.